stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -41,7 +41,122 @@ module SwiftGlibc [system] {
   link "execinfo"
 % end
 
-  header "SwiftGlibc.h"
+% if CMAKE_SDK not in ["ANDROID"]:
+  header "stdc-predef.h"
+  header "features.h"
+% end
+
+  // C standard library
+  header "complex.h"
+  header "ctype.h"
+  header "errno.h"
+  header "fenv.h"
+  header "float.h"
+  header "inttypes.h"
+  header "iso646.h"
+% if CMAKE_SDK not in ["LINUX", "ANDROID"]:
+  header "libutil.h"
+% end
+  header "limits.h"
+  header "locale.h"
+  header "math.h"
+  header "pty.h"
+  header "setjmp.h"
+  header "signal.h"
+  header "stdarg.h"
+  header "stdbool.h"
+  header "stddef.h"
+  header "stdint.h"
+  header "stdio.h"
+  header "stdlib.h"
+  header "string.h"
+  header "tgmath.h"
+  header "time.h"
+% if CMAKE_SDK in ["OPENBSD"]:
+  header "util.h"
+% end
+  header "utmp.h"
+
+  // POSIX
+% if CMAKE_SDK not in ["ANDROID"]:
+  header "aio.h"
+% end
+  header "arpa/inet.h"
+% if CMAKE_SDK not in ["LINUX", "ANDROID"]:
+  header "bsd/ifaddrs.h"
+  header "bsd/pty.h"
+% end
+  header "cpio.h"
+  header "dirent.h"
+  header "dlfcn.h"
+  header "fcntl.h"
+% if CMAKE_SDK not in ["ANDROID"]:
+  header "fmtmsg.h"
+% end
+  header "fnmatch.h"
+  header "ftw.h"
+  header "glob.h"
+  header "grp.h"
+  header "iconv.h"
+  header "ifaddrs.h"
+  header "langinfo.h"
+  header "libgen.h"
+  header "link.h"
+% if CMAKE_SDK not in ["ANDROID"]:
+  header "monetary.h"
+% end
+  header "net/if.h"
+  header "netdb.h"
+  header "netinet/in.h"
+  header "netinet/tcp.h"
+  header "nl_types.h"
+  header "poll.h"
+  header "pthread.h"
+  header "pwd.h"
+  header "regex.h"
+  header "sched.h"
+  header "search.h"
+  header "semaphore.h"
+  header "spawn.h"
+  header "strings.h"
+% if CMAKE_SDK not in ["LINUX", "ANDROID"]:
+  header "sys/event.h"
+% end
+  header "sys/file.h"
+  header "sys/inotify.h"
+  header "sys/ioctl.h"
+  header "sys/ipc.h"
+  header "sys/mman.h"
+  header "sys/mount.h"
+  header "sys/msg.h"
+  header "sys/resource.h"
+  header "sys/select.h"
+  header "sys/sem.h"
+  header "sys/sendfile.h"
+  header "sys/shm.h"
+  header "sys/socket.h"
+  header "sys/stat.h"
+  header "sys/statvfs.h"
+  header "sys/time.h"
+  header "sys/times.h"
+  header "sys/types.h"
+  header "sys/uio.h"
+  header "sys/un.h"
+  header "sys/user.h"
+  header "sys/utsname.h"
+  header "sys/wait.h"
+  header "sysexits.h"
+  header "syslog.h"
+  header "tar.h"
+  header "termios.h"
+  header "unistd.h"
+  header "utime.h"
+  header "wait.h"
+% if CMAKE_SDK not in ["ANDROID"]:
+  header "ulimit.h"
+  header "utmpx.h"
+  header "wordexp.h"
```

Comment:
I tried to keep the ordering of headers the same throughout, in case that matters, but shifted these last two. I can keep them separately above if the order matters.